recipes/carpedeam/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "1.0.0" %}
+{% set sha256 = "92deec3d670810f8433c0345758588b47fa460349e61ce8aa549374f5cf4a812" %}
+
+package:
+  name: carpedeam 
+  version: {{ version|replace("-", ".")  }}
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('carpedeam', max_pin="1.0") }}
```

Comment:
The value should be `"x"` or `"x.x"`, not `"1.0"`.
If you expect to make breaking changes in 1.x then use `"x.x"`

Reply:
Ah sorry, I saw my mistake. Thanks for pushing.